manoseimas/mps_v2/models.py

```diff
@@ -96,7 +97,7 @@ def __unicode__(self):
 class GroupMembership(CrawledItem):
     member = models.ForeignKey(ParliamentMember)
     group = models.ForeignKey(Group)
-    position = models.CharField(max_length=64)
+    position = models.CharField(max_length=128)
     since = models.DateField(blank=True, null=True)
     until = models.DateField(blank=True, null=True)
 
```

Comment:
> On the other hand, your project becomes less modular since you have to provide the same core for each of your apps (modules) to work and it is very hard to avoid inter-module dependencies.

I'm not sure if you are familiar with [Multilayered architecture](http://en.wikipedia.org/wiki/Multilayered_architecture), but it's purpose is to improve modularity. In this case, `core` would be data access layer while other apps would be from domain or business layer. Modules from lower layers can't depend on modules from upper layers and that is how this architecture helps keep modularity in good shape avoiding intermodule dependencies.

But that's just my personal opinion, you can completely ignore it.